workflow/models.py

```diff
@@ -1097,54 +1097,51 @@ class WorkflowLevel2(models.Model):
+    # on_time = models.BooleanField(default=True)
+    # no_explanation = models.TextField("If not on time explain delay", blank=True, null=True)
+    actual_cost = models.DecimalField("Actual Cost", decimal_places=2, max_digits=20, default=Decimal("0.00"), blank=True, help_text="Cost to date calculated from Budget Module")
+    # actual_cost_date = models.DateTimeField(blank=True, null=True)
+    # budget_variance = models.CharField("Budget versus Actual variance", blank=True, null=True, max_length=255)
+    # explanation_of_variance = models.CharField("Explanation of variance", blank=True, null=True, max_length=255)
     total_cost = models.DecimalField("Estimated Budget for Organization", decimal_places=2, max_digits=12, help_text="In USD", default=Decimal("0.00"), blank=True)
-    agency_cost = models.DecimalField("Actual Cost for Organization", decimal_places=2, max_digits=12, help_text="In USD", default=Decimal("0.00"), blank=True)
-    community_handover = models.BooleanField("CommunityHandover/Sustainability Maintenance Plan", help_text='Check box if it was completed', default=False)
-    capacity_built = models.TextField("Describe how sustainability was ensured for this project?", max_length=755, blank=True, null=True)
-    quality_assured = models.TextField("How was quality assured for this project", max_length=755, blank=True, null=True)
-    issues_and_challenges = models.TextField("List any issues or challenges faced (include reasons for delays)", blank=True, null=True)
-    lessons_learned = models.TextField("Lessons learned", blank=True, null=True)
+    # agency_cost = models.DecimalField("Actual Cost for Organization", decimal_places=2, max_digits=12, help_text="In USD", default=Decimal("0.00"), blank=True)
+    # community_handover = models.BooleanField("CommunityHandover/Sustainability Maintenance Plan", help_text='Check box if it was completed', default=False)
+    capacity_built = models.TextField("Describe how sustainability was ensured for this project?", max_length=755, blank=True, null=True, help_text="Descriptive, did this help increases internal or external capacity")
+    quality_assured = models.TextField("How was quality assured for this project", max_length=755, blank=True, null=True, help_text="Descriptive, how was the overall quality assured for this effort")
+    issues_and_challenges = models.TextField("List any issues or challenges faced (include reasons for delays)", blank=True, null=True, help_text="Descriptive, what are some of the issues and challenges")
+    lessons_learned = models.TextField("Lessons learned", blank=True, null=True, help_text="Descriptive, when completed what lessons were learned")
     indicators = models.ManyToManyField("indicators.Indicator", blank=True)
     # sort array for activities related to a project
-    sort = models.IntegerField(default=0, blank=True)
+    # sort = models.IntegerField(default=0, blank=True)
```

Comment:
Remove it

Reply:
These are all fields that I need to confirm with Front-End they are not in use and comments make them stand out so I can share and confirm